client/client.go

```diff
@@ -50,14 +49,13 @@ type Client struct {
 	server neoFSAPIServer
 }
 
-// Init brings the Client instance to its initial state.
+// New creates an instance of Client initialized with the given parameters.
 //
-// One-time method call during application init stage (before Dial) is expected.
-// Calling multiple times leads to undefined behavior.
-//
-// See docs of PrmInit methods for details. See also Dial / Close.
-func (c *Client) Init(prm PrmInit) {
+// See docs of [PrmInit] methods for details. See also [Client.Dial]/[Client.Close].
+func New(prm PrmInit) (*Client, error) {
+	var c = new(Client)
```

Comment:
just a question: do you have a rule why `var c =new` is preferable than `c := new`?